src/frontend/screens/PhotoPreviewModal/index.tsx

```diff
@@ -0,0 +1,497 @@
+                size={20}
+                color={NEW_DARK_GREY}
+                allowFontScaling
+              />
+            }>
+            <BodyText selectable style={styles.primaryInfoText}>
+              {t(m.validatedByCoMapeo)}
+            </BodyText>
+          </InfoItem>
+        )}
+
+        {timestamp !== undefined && (
+          <InfoItem
+            icon={
+              <Octicons
+                name="calendar"
+                size={20}
+                color={NEW_DARK_GREY}
+                allowFontScaling
+              />
+            }>
+            <BodyText selectable style={styles.primaryInfoText}>
+              {formatDate(timestamp, {
+                dateStyle: 'full',
+              })}
+            </BodyText>
+
+            <BodyText selectable style={styles.secondaryInfoText}>
+              {formatTime(timestamp, {
+                timeStyle: 'short',
+              })}
+            </BodyText>
+          </InfoItem>
+        )}
+
+        {coordinates && (
+          <InfoItem
+            icon={
+              <Octicons
+                name="location"
+                size={20}
+                color={NEW_DARK_GREY}
+                allowFontScaling
+              />
+            }>
+            <BodyText selectable style={styles.primaryInfoText}>
+              {formatCoords({
+                lon: coordinates.lon,
+                lat: coordinates.lat,
+                format: coordinateFormat,
+              })}
+            </BodyText>
+          </InfoItem>
+        )}
+
+        {(deviceDetails || cameraDetails || photoDetails) && (
+          <InfoItem
+            icon={
+              <MaterialIcons
+                name="camera"
+                size={20}
+                color={NEW_DARK_GREY}
+                allowFontScaling
+              />
+            }>
+            {deviceDetails && (
+              <BodyText selectable style={styles.primaryInfoText}>
+                {deviceDetails}
+              </BodyText>
+            )}
+
+            {cameraDetails && (
+              <BodyText selectable style={styles.secondaryInfoText}>
+                {cameraDetails}
+              </BodyText>
+            )}
+
+            {photoDetails && (
+              <BodyText selectable style={styles.secondaryInfoText}>
+                {photoDetails}
+              </BodyText>
+            )}
+          </InfoItem>
+        )}
+
+        {observationDocId && (
+          <InfoItem
+            icon={
+              <MaterialIcons
+                name="numbers"
+                size={20}
+                color={NEW_DARK_GREY}
+                allowFontScaling
+              />
+            }>
+            <BodyText selectable style={styles.primaryInfoText}>
+              {
+                // TODO: Showing this ID is a temporary measure
+                // Ideally we use a format that is more human readable/friendly
+                observationDocId.slice(0, 15)
```

Comment:
decision to truncate was based on Gregor's recommendation as a temporary measure.